map/map_loader/src/lanelet2_map_loader/lanelet2_map_loader_node.cpp

```diff
@@ -63,6 +82,22 @@ Lanelet2MapLoaderNode::Lanelet2MapLoaderNode(const rclcpp::NodeOptions & options
 
   declare_parameter("lanelet2_map_path", "");
   declare_parameter("center_line_resolution", 5.0);
+
+  std::vector<std::string> tmp_lanelet2_paths_or_directory =
+    declare_parameter<std::vector<std::string>>("lanelet2_map_folder_path");
+  std::string tmp_lanelet2_map_metadata_path =
+    declare_parameter<std::string>("lanelet2_map_metadata_path");
```

Comment:
I want to make sure that users can load a single lanelet2 map as before (without setting new parameters) so we could put these declare parameters only when `enable_dynamic_lanelet_loading` is true

Reply:
I put these two declaration on if statement. 👍👍

In current approach I did not change the whole lanelet2 map loading parts, so it can load whole lanelet2 same as before if we use dynamic lanelet2 map loading.